aws/app/ecs_iam.tf

```diff
@@ -106,6 +106,42 @@ resource "aws_iam_role_policy_attachment" "dynamodb_forms" {
   policy_arn = aws_iam_policy.forms_dynamodb.arn
 }
 
+resource "aws_iam_role_policy_attachment" "sqs_forms" {
+  role       = aws_iam_role.forms.name
+  policy_arn = aws_iam_policy.forms_sqs.arn
+}
+
+#
+# IAM - SQS
+#
+
+resource "aws_iam_policy" "forms_sqs" {
+  name        = "forms_sqs"
+  path        = "/"
+  description = "IAM policy to allow access to SQS for Forms ECS task"
+  policy      = data.aws_iam_policy_document.forms_sqs.json
+
+  tags = {
+    (var.billing_tag_key) = var.billing_tag_value
+    Terraform             = true
+  }
+}
+
+data "aws_iam_policy_document" "forms_sqs" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "sqs:GetQueueUrl",
+      "sqs:SendMessage"
+    ]
+
+    resources = [
+      var.sqs_reliability_queue_arn
```

Comment:
Maybe I'm wrong here but it seems this will only grant permission to the reliability queue however we are using a different queue which is the reprocessing queue

Reply:
Nice catch! I originally was using the same reliability queue 😆